### PR TITLE
docs(Card): move card view to demos section

### LINKED ISSUE #5599: Move Card View demo from Card to a new page within Demos
Want to move the Card View demo from the Card component page to a new page that lives in the Demos section. This will group them better with other similar demos (like primary-detail) and reserve Card demos for examples of specific card types. New page should have ID: Card view and section: demos.

### PR DESCRIPTION
**Pull request by kmcfaul**
**What**: Closes #5599

### CODE REVIEW

**Review by mcarrano — CHANGES_REQUESTED** [review]
Card view is not relocated into Demos, as expected. However when I open the demo, all of the card content is missing at https://patternfly-react-pr-5671.surge.sh/demos/card-view/react-demos/card-view/

Here's what I see...

Screen Shot 2021-04-15 at 10 15 53 AM

**Comment by kmcfaul** [comment]
It appears the `fetch` used to populate the cards is taking very long, when I left the page open for a bit it eventually populated @mcarrano 

When I test locally, the fetch was blocked by an error, but that doesn't seem to be the case for the surge, it's just slow.

File: packages/react-core/src/demos/Card/Card.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-id: Card
-section: components
+id: Card view
+section: demos
```
Comment (redallen) [inline packages/react-core/src/demos/Card/Card.md:3]: Please update the screenshots: `yarn build:docs && yarn serve:docs` then in a separate terminal `yarn screenshots`.

**Review by redallen — CHANGES_REQUESTED** [review]

File: packages/react-core/src/demos/Card/Card.md (hunk 1 above)
Comment (kmcfaul) [reply packages/react-core/src/demos/Card/Card.md:3]: I'm getting an error with the `screenshots` command, says it doesn't find the command?

Comment (redallen) [reply packages/react-core/src/demos/Card/Card.md:3]: Ah sorry, try `yarn workspace @patternfly/react-docs screenshots`. Please add `"screenshots": "yarn workspace @patternfly/react-docs screenshots",` so you just have to run `yarn screenshots` in the future!

**Comment by mcarrano** [comment]
> It appears the fetch used to populate the cards is taking very long, when I left the page open for a bit it eventually populated @mcarrano

Seems to be working now

File: packages/react-core/src/demos/Card/Card.md (hunk 1 above)
Comment (kmcfaul) [reply packages/react-core/src/demos/Card/Card.md:3]: Thanks that did the trick! Updated.

Comment (kmcfaul) [reply packages/react-core/src/demos/Card/Card.md:3]: Added the command to the package.json as well

**Comment by kmcfaul** [comment]
@redallen Anything I can do the resolve that a11y error on my end? It looks like it didn't run.

**Comment by redallen** [comment]
@kmcfaul You can rekick the job once it finishes running: 
image

I've never seen that specific CircleCI error before.

**Review by nicolethoen — APPROVED** [review]

**Review by mcarrano — APPROVED** [review]
Looks good now. Thanks @kmcfaul .

**Review by mcoker — APPROVED** [review]
LGTM

**Review by dlabrecq — APPROVED** [review]

**Review by redallen — APPROVED** [review]
Thanks @kmcfaul !

**Pull request merged**